staff/irene-roldan/strings/concat.js

```diff
@@ -0,0 +1,17 @@
+delete String.prototype.concat()
+
+function concat (element1, element2, separator){
+    //crear una variable auxiliar 'result' que sume valores de element + separator
+    var result = element1 + separator + element2
+    //función que devuelva variable auxiliar 'result'
+    return result
+    
+}
+
```

Comment:
What happens ir we want to concatenate more than 2 elements? Check the concat() behaviour when we use it like this example: 

```
str1.concat(str2, " ", str1, " ",str2)
```